debian/ubuntu-core-launcher.postinst

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+
+# XXX: doesn't seem to really work in practice. Perhaps the version needs love
+dpkg-maintscript-helper rm_conffile \
```

Comment:
Instead of writing this in three places (postinst, postrm, preinst), it can also be written in a single place: `debian/snap-run.mainscript` with the content `rm_conffile /etc/apparmor.d/usr.bin.ubuntu-core-launcher 2.0~`.